data/scripts/molgenis-model-update/update/update_4_x.py

```diff
@@ -0,0 +1,205 @@
+import shutil
+import os
+import pandas as pd
+
+
+def float_to_int(df):
+    """
+    Cast float64 Series to Int64.
+    """
+    for column in df.columns:
+        if df[column].dtype == 'float64':
+            df.loc[:, column] = df[column].astype('Int64')
+
+    return df
+
+
+def get_data_model(profile_path, path_to_write, profile):
+    # get data model from profile and write to file
+    data_model = pd.DataFrame()
+
+    for file_name in os.listdir(profile_path):
+        if '.csv' in file_name:
+            df = pd.read_csv(profile_path + file_name)
+            df = df.loc[df['profiles'].str.contains(profile)]
+            data_model = pd.concat([data_model, df])
+
+    data_model = float_to_int(data_model)
+    data_model.to_csv(path_to_write, index=None)
+
+
+class Transform:
+    """General functions to update catalogue data model.
+    """
+
+    def __init__(self, database_name, database_type):
+        self.database_name = database_name
+        self.database_type = database_type
+        self.path = self.database_name + '_data/'
+
+    def delete_data_model_file(self):
+        """Delete molgenis.csv
+        """
+        os.remove(self.path + 'molgenis.csv')
+
+    def update_data_model_file(self):
+        """Get path to data model file and copy molgenis.csv to appropriate folder if it does not exist
+        """
+        # get molgenis.csv location
+        if self.database_type == 'catalogue':
+            data_model = os.path.abspath('../../../datacatalogue/molgenis.csv')
+            profile_path = os.path.abspath('../../../_models/shared/')
```

Comment:
On iOS the last "/" is not included in the profile_path although it's in the input, which results in errors when reading filenames. It turns out this is a known issue: https://github.com/florianschanda/miss_hit/issues/237
For testing I've therefore included an extra line:
`profile_path = os.path.abspath('../../../_models/shared')`
`profile_path = profile_path + '/'`

Reply:
I have included pathlib specification of the path, which should solve the issue.